### DISCUSSION TITLE
JavaScript Snippet Includes

### DISCUSSION THREAD

**Discussion by strickvl** [quarto-dev/quarto-cli#3818] [Q&A] [answered]
I'm trying to add analytics to my website. I use [Plausible](https://plausible.io/) and the scripts come in a shape that [looks like this](https://plausible.io/docs/plausible-script):

```JavaScript
<script defer data-domain="yourdomain.com" src="https://plausible.io/js/script.js"></script>
```

Instructions state:

```
Paste your Plausible Analytics tracking script code into the Header (<head>) section of your site. Place the tracking script within the <head> … </head> tags.
```

So I gather I should be doing something with `includes`, but I can't for the life of me figure out how to do this on a site basis. I can't actually even do this on a single page.

I note that the Quarto docs seem to have previously had a section on including custom scripts (at least, assuming that from the TOC for the page) but that no longer is specified [in the docs themselves](https://quarto.org/docs/websites/website-tools.html#custom-scripts-and-cookie-consent).

I imagine the solution to this is a single line. Can someone help me find that line and where to put it? In case it helps, the repo for my quarto website is public and available [here](https://github.com/strickvl/mlops-dot-systems).

**Comment by dragonstyle** [comment, marked as answer]
I think you're looking for this option:

https://quarto.org/docs/output-formats/html-basics.html#includes

You can use this in your _quarto.yml file like so:

```
format:
  html:
    include-in-header:
      - <file path>
```

or

```
format:
  html:
    include-in-header:
      text: |
        <script><\script>
```

Hth!

**Reply by strickvl** [reply]
Thank you! That did the trick.
